packages/cursorless-vscode-e2e/suite/fixtures/recorded/languages/perl/changeItem.yml

```
@@ -0,0 +1,30 @@
+languageId: perl
+command:
+  version: 4
+  spokenForm: change item
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: collectionItem}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |-
+    %hash = (
+        one => 1,
+        two => 2,
+    );
+  selections:
+    - anchor: {line: 1, character: 7}
+      active: {line: 1, character: 7}
+  marks: {}
+finalState:
+  documentContents: |-
+    %hash = (
+        one => ,
```

Item should be the whole thing.

```suggestion
        ,
```

What you have here is "value"

Also worth a "chuck item" test to make sure the `,` gets cleaned up

---

I've changed the test (now failing as the behaviour is not yet fixed) and added a chuck item test (also failing).

---

This is now fixed